Scala/Scala.sublime-syntax

```diff
@@ -7,6 +7,9 @@ file_extensions:
   - sbt
 scope: source.scala
 variables:
+  # lookahead/behind for operators
+  oplookbehind: '(?<=[[[:alpha:]]0-9\s\(\)\[\]\{\}])'
```

Comment:
Backtracking is generally bad for performance.

Instead, consume the `::` and the `==>` with a match rule. Then the second `:` and the trailing `=>` won't be possible to be matched against.